TCL4 Data Management/utm-tcl4-dmp-fet-latency.yaml

```diff
@@ -88,7 +88,7 @@ definitions:
     properties:
       metaData:
         $ref: 'https://raw.githubusercontent.com/nasa/utm-docs/develop/TCL4%20Data%20Management/utm-tcl4-dmp-common.yaml#/definitions/metaData'
-      sensorLatencyTupleArray:
+      sensorLatencyTuple:
```

Comment:
I think we should change the required not the field

Reply:
We had a brief email/slack discussion about this a while back with Priya & David.

The DMP rev 3 maps these items to the specific elements using "sensorLatencyTuple". If we change the required instead of the field, that mapping is no longer valid. Would require another DMP version to change this.

The "Array" is also misleading because it isn't an array.